src/rad/resources/schemas/guidewindow-1.0.0.yaml

```
@@ -179,21 +179,21 @@ properties:
   pedestal_frames:
     title: "Reconstituted and oriented pedestal frame GW images. Dimensions: num_frames,
             num_combined_resultants (or num_uncombined_resultants), num_reads, x, y"
-    tag: tag:stsci.edu:asdf/core/ndarray-1.0.0
-    datatype: uint16
-    ndim: 5
+    tag: tag:stsci.edu:asdf/unit/quantity-1.1.0
+    #datatype: uint16
+    #ndim: 5
```

```suggestion
    tag: tag:stsci.edu:asdf/unit/quantity-1.1.0
      value:
        tag: tag:stsci.edu:asdf/core/ndarray-1.0.0
        datatype: uint16
        ndim: 5
```

I think this is how you can specify the information inside the quantity tag rather than just commenting it.

---

I'm not sure if my comment is exactly right. Let me know if it works. If it does not, there is some combo of things that I can spend a bit of time figuring out for you.

---

This is resolved with William's work on asdf-astropy and his implementation of the changes above.